lib/components/victory-area.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Dimensions, Platform } from "react-native";
+import { VictoryLabel, VictoryContainer, Area, VictoryClipContainer } from "victory-core-native";
```

Comment:
@sibelius sorry about that. i've added the WIP tag on the PR, as it is a little half-baked at the moment